src/main/resources/jupyter/cluster-docker-compose.yaml

```diff
@@ -1,33 +1,36 @@
-proxy:
-  container_name: "${PROXY_SERVER_NAME}"
-  image: "${PROXY_DOCKER_IMAGE}"
-  net: host
-  volumes:
-    - /certs/jupyter-server.crt:/etc/ssl/certs/server.crt:ro
-    - /certs/jupyter-server.key:/etc/ssl/private/server.key:ro
-    - /certs/rootCA.pem:/etc/ssl/certs/ca-bundle.crt:ro
-    - /etc/cluster-site.conf:/etc/apache2/sites-enabled/site.conf
-    - /auth_openidc.conf:/etc/apache2/mods-enabled/auth_openidc.conf
-  restart: always
-  environment:
-    HTTPD_PORT: '80'
-    SSL_HTTPD_PORT: '443'
+version: '2'
+services:
+  proxy:
+    container_name: "${PROXY_SERVER_NAME}"
+    image: "${PROXY_DOCKER_IMAGE}"
+    network_mode: host
```

Comment:
changes to this compose file because of the new version of compose:
- added `version` at the top
- services now go under `services`
- `net` is now called `network_mode`